main.js

```diff
@@ -177,6 +178,22 @@ document.getElementById("start").onclick = function() {
   start();
 };
 
+document.getElementById("canvasSize").onchange = function() {
+  Array.prototype.forEach.call(document.body.classList, function(c) {
+    if (c.indexOf('size-') == 0) {
+      document.body.classList.remove(c);
+    }
+  });
+
+  if (this.value) {
+    document.body.classList.add(this.value);
+  }
+
+  MIDP.updatePhysicalScreenSize();
+  MIDP.updateCanvas();
+  start();
```

Comment:
Here are some screenshots that show what happens. This is what I saw when switching from 240x320 to 480x320 while running unit tests:

![screen shot 2015-03-12 at 17 54 09](https://cloud.githubusercontent.com/assets/305455/6631315/eff83980-c8e0-11e4-80e7-aa222fbfc64e.png)

And then after switching to 320x240:

![screen shot 2015-03-12 at 17 54 20](https://cloud.githubusercontent.com/assets/305455/6631325/1060a5f4-c8e1-11e4-93bd-2fe506161e66.png)